tests/ashley/lti_utils.py

```diff
@@ -0,0 +1,47 @@
+"""This package contains helper functions to build LTI requests"""
+
+from urllib.parse import unquote
+
+from oauthlib import oauth1
+
+CONTENT_TYPE = "application/x-www-form-urlencoded"
+
+
+def sign_parameters(passport, lti_parameters, url):
```

Comment:
could also be part of lti toolbox lib in the future...?

Reply:
In fact, it is part of lti toolbox : https://github.com/openfun/django-lti-toolbox/blob/master/tests/lti_toolbox/utils.py

But since it's in the `tests` namespace, it's not bundled with the package on pypi.
I'll take it if you have good practices to share on this subject.